### ISSUE TITLE
Broken <docs> in Vue SFC

### ISSUE THREAD

**Issue by pribilinskiy** [tuchk4/storybook-readme#109]
[The readme lacks an example](https://github.com/tuchk4/storybook-readme#webpack-configuration-for-vue-storybook) on how to use `<docs>` section in Vue SFC files.
So I've found it in sources:
https://github.com/tuchk4/storybook-readme/blob/e2793069d0770fbca3c8fd5c759ce72fa4b21b67/packages/example-vue/stories/index.js#L44

And even in that case it doesn't work with the following code:
```js
  // eslint-disable-next-line no-underscore-dangle
  .addDecorator(withDocs(MyComponent.__docs))
```

I get this error in preview area:

![image](https://user-images.githubusercontent.com/3078595/50991720-0ba3d800-151e-11e9-8f85-ff11feb13e13.png)

webpack config contains the `blockType=docs` rule

Any ideas what may be the cause?

**Comment by tuchk4** [comment]
Here is example of vue-storybook  https://github.com/tuchk4/storybook-readme/tree/master/packages/example-vue with `<docs>` but I am not sure if it works with latest versions.

Will check it right now

**Comment by tuchk4** [comment]
This example https://github.com/tuchk4/storybook-readme/tree/master/packages/example-vue  works correctly.

You  may got error because `MyComponent.__docs` is undefined. Seems something with webpack config.

Could you please publish repo with reproduction?

**Comment by pribilinskiy** [comment]
Will do for sure once return to our storybook feature within a month. Thanks

**Comment by yoyoys** [comment]
use `MyComponent.options.__docs`

**Comment by OwenMelbz** [comment]
@tuchk4 Having the same issue now :(

When we run storybook in the watch/serve mode it works fine e.g.

```js
storiesOf('Testing', module)
.addDecorator(withDocs(MyButton.__docs))
```

with a webpack.config.js of 

```
module.exports = storybookBaseConfig => {
    storybookBaseConfig.module.rules.push({
        resourceQuery: /blockType=docs/,
        use: [
            'storybook-readme/env/vue/docs-loader',
            'html-loader',
            'markdown-loader',
        ],
    })

    return storybookBaseConfig
}
```
--

Everything is well and good!

However.......................

When you run the build process the `MyButton.__docs` is undefined causing the error

```
storybook-readme: wrong arguments
```

**Comment by yoyoys** [comment]
@owenmelbz `MyButton.options.__docs` Should be useful

**Comment by OwenMelbz** [comment]
`MyButton.options` is also always empty

**Comment by b0g3r** [comment]
FYI @tuchk4 @yoyoys @OwenMelbz 
My solution, just extend options definition of component with typescript: 
https://github.com/vuejs/vue-class-component/issues/50#issuecomment-464508801

**Comment by OwenMelbz** [comment]
@b0g3r and if your not using typescript?

**Comment by b0g3r** [comment]
Do you have reproduced example? Do you use vue-property-decorator?

**Comment by pribilinskiy** [comment]
Yes, we use `vue-property-decorator` in every component

**Comment by tuchk4** [comment]
Going to make new release https://github.com/tuchk4/storybook-readme/issues/118
Hope will be possible to fix.

@OwenMelbz Errors only during build?  There are no errors when run storybook?

**Comment by OwenMelbz** [comment]
Nope, when storybook is running in server mode it’s auto-reloading and everything is working as expected.

Only when we run build does it throw the error

**Comment by tuchk4** [comment]
Hope it is fixed in `5.0.0`
Made test for dev and pro builds.

**Comment by tuchk4** [comment]
```js
module.exports = storybookBaseConfig => {
  storybookBaseConfig.module.rules.push({
    resourceQuery: /blockType=docs/,
    use: ['storybook-readme/vue/docs-loader', 'html-loader', 'markdown-loader'],
  });
};
```

Define `<docs>` tag inside vue module:

```vue
<docs>
Docs inside vue module 
</docs>

<template>
  <button class="button">
    <slot></slot>
  </button>
</template>
```

Use it to defone docs at story:

```js
import MyButton from '../components/MyButton/MyButton.vue';

storiesOf('Vue <docs>', module).addParameters({
  readme: {
    content: MyButton.__docs,
  },
});
```
